src/core/tests/model.cpp

```diff
@@ -1147,6 +1147,8 @@ TEST(model, add_output_port_to_result) {
 }
 
 TEST(model, add_output_performance) {
+    // skip this test due to CVS-140440
+    GTEST_SKIP();
```

Comment:
```suggestion
    SKIP_IF_CURRENT_TEST_IS_DISABLED();
```

Reply:
SKIP_IF_CURRENT_TEST_IS_DISABLED is defined in func test utils.
I don't want to add dependency to func test utils to unit test target.

Reply:
```suggestion
    GTEST_SKIP() << "CVS-140440";
```